delivery.yaml

```diff
@@ -0,0 +1,42 @@
+build_steps:
+    - desc: 'Install required build software'
+      cmd: |
+        apt-get install -y make git apt-transport-https ca-certificates curl
+    - desc: 'Install go'
+      cmd: |
+        add-apt-repository -y ppa:longsleep/golang-backports
+        apt-get update
+        apt-get install -y golang-go
+    - desc: 'Install Docker'
+      cmd: |
+        # enable experimental features (--squash)
+        mkdir -p /etc/systemd/system/docker.service.d
+        cat << EOF > /etc/systemd/system/docker.service.d/docker.conf
+        [Service]
+        ExecStart=
+        ExecStart=/usr/bin/dockerd -H fd:// --experimental=true
+        EOF
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+        apt-key fingerprint 0EBFCD88
+        add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+        apt-get update
+        apt-get install -y docker-ce
+    - desc: 'Symlink sources into the GOPATH'
+      cmd: |
+        export GOPATH=$HOME/go
+        export TOP_DIR=$GOPATH/src/github.com/zalando
+        mkdir -p $TOP_DIR
+        ln -s $(pwd) $TOP_DIR/skipper
+    - desc: 'Build & push docker image'
+      cmd: |
+        export PATH=$PATH:$HOME/go/bin
+        IS_PR_BUILD=${CDP_PULL_REQUEST_NUMBER+"true"}
+        if [[ ${CDP_TARGET_BRANCH} == "master" && ${IS_PR_BUILD} != "true" ]]
+        then
+          IMAGE=registry-write.opensource.zalan.do/pathfinder/skipper
+        else
+          IMAGE=registry-write.opensource.zalan.do/pathfinder/skipper-test
+        fi
```

Comment:
Great idea with the `-test` suffix :+1: , I think we should adopt this everywhere if the build pushes on PRs.